docs/components/edge/edged.md

```diff
@@ -66,58 +66,61 @@ Currently kubeedge edged supports only docker runtime using the legacy dockertoo
 
 ## Secret Management
 
-At edged, Secrets are handled separately. For its operations like addition, deletion and modifications; there are separate set of config messages or interfaces.
+At edged, Secrets are handled separately. For operations like addition, deletion and modifications, there are separate sets of config messages or interfaces.
 Using these interfaces, secrets are updated in cache store.
-Below flow diagram explains the message flow.
+The flow diagram below explains the message flow.
 
 ![Secret Message Handling](../../images/edged/secret-handling.png)
 
 *Fig 7: Secret Message Handling at EdgeD*
 
-Also edged uses MetaClient module to fetch secret from Metamanager (if available with it) else cloud. Whenever edged queries for a new secret which Metamanager doesn't has, the request is forwared to cloud. Before sending the response containing the secret, it stores a copy of it and send it to edged.
-Hence the subsequent query for same secret key will be responded by Metamanger only, hence reducing the response delay.
-Below flow diagram shows, how secret is fetched from metamanager and cloud. The flow of how secret is saved in metamanager.
+Also edged uses the MetaClient module to fetch secrets from Metamanager (if available), falling back to cloud. Whenever edged queries for a new secret which Metamanager doesn't have, the request is forwarded to cloud. Before sending the response containing the secret, it stores a copy of it and sends it to edged.
```

Comment:
: ), what does `falling back to cloud` here mean?

Reply:
@GsssC good question -- in my first pass I tried to keep the wording fairly similar to the original, but you're right this still isn't very clear.  I gave this paragraph and a similar paragraph on ConfigMaps another pass in f494caf based on the diagrams (which are very helpful!).